infrastructure/openhim-base-config.json

```diff
@@ -1506,6 +1506,20 @@
           "primary": false,
           "username": "",
           "password": ""
+        },
+        {
+          "type": "http",
+          "status": "enabled",
+          "forwardAuthHeader": true,
+          "name": "Webhooks -> Death Registration",
+          "secured": false,
+          "host": "webhooks",
+          "port": 2525,
+          "path": "",
+          "pathTransform": "",
+          "primary": false,
+          "username": "",
+          "password": ""
```

Comment:
We would probably need to move this to a migration

Reply:
Done :)